pkg/plugins/backendplugin/grpcplugin/client_v2.go

```diff
@@ -356,15 +356,14 @@ func handleGrpcStatusError(ctx context.Context, errorSource errstatus.Source, er
 		if innerErr != nil {
 			logger.Error("Could not set downstream error source", "error", innerErr)
 		}
-		return plugins.ErrPluginDownstreamErrorBase.Errorf("%v", err)
+		return plugins.ErrPluginRequestFailureErrorBase.Errorf("%v", err)
```

Comment:
nit: the comment for this function mentions `a plugin downstream error is returned as both plugin and downstream errors are treated the same in Grafana.`, maybe we should look at that wording as well?

Reply:
That one is actually correct. The plugin returns an error source, which can be either the "plugin" or a "downstream" and we log it. However, this error base was only intended to indicate that the error originated from the plugin. It used the same wording, which made it slightly confusing.